phpstan.neon.dist

```diff
@@ -0,0 +1,14 @@
+includes:
+    - vendor/szepeviktor/phpstan-wordpress/extension.neon
+parameters:
+    level: max
+    autoload_files:
+        - wp-graphql.php
+        - access-functions.php
```

Comment:
I tried to run this locally and it bombs. 

I think the autoload_files needs to change to 

```yaml
autoload_files:
        - vendor/autoload.php
```

Reply:
Not sure what I'm doing wrong, but as long as I have the following, I can't get this to run at all: 

```yaml
includes:
    - vendor/szepeviktor/phpstan-wordpress/extension.neon
```

Reply:
Ah, I had to run with `--memory-limit=-1` and now it works for me